src/protocol/mod.rs

```diff
@@ -51,6 +54,7 @@ impl Default for WebSocketConfig {
             max_send_queue: None,
             max_message_size: Some(64 << 20),
             max_frame_size: Some(16 << 20),
+            max_redirects: Some(3)
```

Comment:
Unless tokio-tungstenite uses the connect functions instead of the client functions to build its API the redirects specified here won't make a difference, since it'd need to implement its own redirect code where the config could be handled.